client/my-sites/earn/home.tsx

```diff
@@ -0,0 +1,196 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import React, { FunctionComponent } from 'react';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+import { SiteSlug } from 'types';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+import getSiteBySlug from 'state/sites/selectors/get-site-by-slug';
+import { getCurrentPlan } from 'state/sites/plans/selectors';
+import PromoSection, { Props as PromoSectionProps } from 'components/promo-section';
+import {
+	FEATURE_WORDADS_INSTANT,
+	FEATURE_SIMPLE_PAYMENTS,
+	FEATURE_UPLOAD_PLUGINS,
+	FEATURE_NO_ADS,
+} from 'lib/plans/constants';
+
+interface ConnectedProps {
+	selectedSiteSlug: SiteSlug;
+	currentPlan: string;
+}
+
+const Home: FunctionComponent< ConnectedProps > = ( { selectedSiteSlug } ) => {
+	const translate = useTranslate();
+
+	/**
+	 * Return the content to display in the Simple Payments card based on the current plan.
+	 *
+	 * @returns {object} Object with props to render a PromoCard.
+	 */
+	const getSimplePaymentsCard = () => {
+		// Todo: get plan and return different content based on it.
+
+		const supportLink =
+			'https://en.support.wordpress.com/wordpress-editor/blocks/simple-payments-block/';
+		return {
+			title: translate( 'Collect one-time payments' ),
+			body: translate(
+				'Add a payment button to any post or page to collect PayPal payments for physical products, digital goods, services, or donations. {{em}}Available to any site with a Premium plan{{/em}}.',
+				{
+					components: {
+						em: <em />,
+					},
+				}
+			),
+			image: {
+				path: '/calypso/images/earn/simple-payments.svg',
+			},
+			cta: {
+				feature: FEATURE_SIMPLE_PAYMENTS,
+				upgradeButton: {
+					text: translate( 'Upgrade to Premium Plan' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Upgrade to a Premium Plan!' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).